### DISCUSSION TITLE
v1.2.3-beta12

### DISCUSSION THREAD

**Discussion by constanline** [constanline/XQuickEnergy#474] [Announcements/公告]
## What's Changed
* 添加Hook支付宝推荐应用默认勾选 by @nov30th in https://github.com/constanline/XQuickEnergy/pull/435
* 收取复活好友回赠的金球 by @acooler15 in https://github.com/constanline/XQuickEnergy/pull/448
* 增加“多选”和“反选” by @acooler15 in https://github.com/constanline/XQuickEnergy/pull/452
* 合并来自cwj851的pr by @cwj851 in https://github.com/constanline/XQuickEnergy/pull/471

## New Contributors
* @acooler15 made their first contribution in https://github.com/constanline/XQuickEnergy/pull/448

**Full Changelog**: https://github.com/constanline/XQuickEnergy/compare/v1.2.3-beta6...v1.2.3-beta12

<hr /><em>This discussion was created from the release <a href='https://github.com/constanline/XQuickEnergy/releases/tag/v1.2.3-beta12'>v1.2.3-beta12</a>.</em>

**Comment by xzcgg** [comment]
开发者大大，可以开放光盘行动吗🥺🥺🥺

**Comment by 313is** [comment]
弄个赞助吧，开发者大哥

**Reply by Zhangfanyu1222** [reply]
![图片](https://github.com/constanline/XQuickEnergy/assets/65014915/2b5dbadf-54e1-4ca1-8222-7a3bac97615b)
app/src/main/res/drawable/donation.jpg这张图就是

**Comment by emmming** [comment]
作者什么时候再加个雇佣小鸡跟加好友

**Comment by Gu888** [comment]


**Reply by acooler15** [reply]
我的一直没问题？你开启小鸡日记了吗？

**Comment by OoLaLa-yy** [comment]
请问最高支持到哪个版本？不升新版又不能集福

**Reply by Zhangfanyu1222** [reply]
支持最新版呀，可以升级集五福

**Reply by OoLaLa-yy** [reply]
v1.2.3-beta6那会，装最新版，提示没文件访问权限的，吓得装回旧版

**Reply by Zhangfanyu1222** [reply]
安装支付宝和芝麻粒后不启动，在Download目录下创建一个useMedia文件夹，在Android/media文件夹下创建com.eg.android.AlipayGphone文件夹，最后把Download目录下的xqe文件夹复制到刚刚新创建的com.eg.android.AlipayGphone文件夹里

**Reply by OoLaLa-yy** [reply]
感谢大佬，按此操作在01/18更新的v10.5.66.8000使用成功

**Reply by OoLaLa-yy** [reply]
试了2遍，同样的操作在miui14.0.10失败，支付宝能正常启动，但芝麻粒没反应，通知栏也不显示，请问miui比较奇怪吗？

**Reply by Zhangfanyu1222** [reply]
通知栏需要 森林或者庄园的主开关打开了 才会显示

**Reply by OoLaLa-yy** [reply]
useMedia文件夹删了+重启=记录、统计都没了、芝麻粒仍然无效果（PS.之前成功的系统是ColorOS_v13.1.0.183(CN01)F.30）看来miui是没救得回退芝麻粒版本了...

**Reply by doooq** [reply]
> 安装支付宝和芝麻粒后不启动，在Download目录下创建一个useMedia文件夹，在Android/media文件夹下创建com.eg.android.AlipayGphone文件夹，最后把Download目录下的xqe文件夹复制到刚刚新创建的com.eg.android.AlipayGphone文件夹里

大佬，Download里面没有xqe文件

**Comment by mrlihx** [comment]
双开的支付宝 启动白屏，原始的正常。

**Comment by lirui758** [comment]
神奇海洋可以添加15s的任务吗…潘多拉的任务每次都需要自手动领取
